tic_tac_toe/game.py

```diff
@@ -87,18 +147,60 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    #create board object
+    board = game['board']
+    #check illegal moves
+    if game['winner'] or _board_is_full(board):
+        raise InvalidMovement('Game is over.')
+    if player != game['next_turn']:
+        raise InvalidMovement('"{}" moves next.'.format(game['next_turn']))
+    if not _position_is_valid(position):
+        raise InvalidMovement('Position out of range.')
+    if not _position_is_empty_in_board(position, board):
+        raise InvalidMovement('Position already taken.')
+    #make actual move
+    board[position[0]][position[1]] = player
+    #set winner
+    winner = _check_winning_combinations(board, player)
+    #explore options if a winner exists
+    if winner:
+        game['winner'] = winner
+        game['next_turn'] = None
```

Comment:
You don't need to set `next_turn` to `None` as your code never actually checks to see if it is `None`, it only checks if `next_turn` matches `player`.